libs/eo/shared/services/src/api-versioning/api-versioning.interceptor.ts

```diff
@@ -0,0 +1,33 @@
+
+import { Inject, Injectable } from '@angular/core';
+import { HttpInterceptor, HttpRequest, HttpHandler, HttpEvent, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { EoApiEnvironment, eoApiEnvironmentToken } from '@energinet-datahub/eo/shared/environments';
+
+@Injectable()
+export class EoApiVersioningInterceptor implements HttpInterceptor {
+  constructor(@Inject(eoApiEnvironmentToken) private apiEnvironment: EoApiEnvironment) {}
+
+  intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
+    const apiVersions = this.apiEnvironment.apiVersions;
+    const versionedPaths = Object.keys(apiVersions);
+    const isVersioned = versionedPaths.find((path) => request.url.includes(path));
```

Comment:
This looks to me that it takjes the request url to see which sub system to use.
What if we change the urls to not include the sub systems?